package.json

```diff
@@ -52,6 +52,7 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
+    "@discordjs/builders": "^1.1.0",
```

Comment:
https://discordjs.guide/additional-info/changes-in-v14.html#builders-are-now-included-in-v14
```suggestion
```

Reply:
Bien vu j'avais carrement oublié sa  🤯